index.js

```diff
@@ -149,8 +151,8 @@ function makeCachedResolver (prefix, fieldName, cache, originalFieldResolver, po
         result = await originalFieldResolver(self, arg, ctx, info)
       } else {
         // use cache to get the result
-        // note the cache function never throws error
-        result = await cache[name]({ self, arg, ctx, info })
+        // Ignore execptions, 'onError' is already in place
```

Comment:
this comment is not true, looks like the function does not throw exception because onError is defined, but the function does not throw any exception anyway

Reply:
I believe it does throw, Simone. At least I saw it happening when the resolver throws an exception. 

You can see an example on [this test](https://github.com/mercurius-js/cache/blob/072b6fb0b666c8f6775ac047a0f37599d973d796/test/base.test.js#L1119), try enclosing the `await cache` in a try-catch, it is going to catch the exception.

This behavior is shown also on [this test in async-cache-dedupe](https://github.com/mcollina/async-cache-dedupe/blob/16464b635abc5a937c8bc403ab2e401fbabb25f2/test/base.test.js#L554), where we can see the exception being thrown on the `await` alongside the promise being rejected. 

I believe this happens because of the way [buildPromise](https://github.com/mcollina/async-cache-dedupe/blob/16464b635abc5a937c8bc403ab2e401fbabb25f2/src/cache.js#L245) is implemented, it is expected to throw an exception to the upper caller and also to be rejected and handled on the `query.promise.catch`.

Reply:
sorry, my bad.
I meant, the exception is already managed, because it's inside a try-catch, not because `onError` is defined
but you are right, the comment is not appropriate, so please remove them it

Reply:
No problem!